vignettes/f

```diff
@@ -0,0 +1,11 @@
+"oldValue","newValue"
```

Comment:
This temporary file is created as a side effect of the vignette. We should either a) clean this up, via a not displayed code chunk that unlinks `f` or b) set the `IDrecode` chunck to `eval = FALSE`. I would suggest b), but both are OK with me.